app/api/helpers/permission_manager.py

```diff
@@ -154,9 +154,13 @@ def is_speaker_for_session(view, view_args, view_kwargs, *args, **kwargs):
         return view(*view_args, **view_kwargs)
 
     if session.speakers:
+        # If the session is being edited
         for speaker in session.speakers:
             if speaker.user_id == user.id:
                 return view(*view_args, **view_kwargs)
+    else:
+        # If the session is being created
+        return view(*view_args, **view_kwargs)
```

Comment:
This will allow access for all other cases

Reply:
@iamareebjamal only in the sessions which have no speakers. but to register a session, we need to have at least one speaker. right?

Reply:
Confirm if this is true

Reply:
@iamareebjamal I will try and make a session via api and get back to you

Reply:
@iamareebjamal No, my observations were incorrect. I will come up with another solution in that case.